### PR TITLE
Qna080521

### PR DESCRIPTION
**Pull request by a-nouraldean**
@PBandJaimie @Taniel-P @EJLEE808 Added two small components

### CODE REVIEW

File: client/src/components/QandA/QandA.jsx [hunk 1]
```diff
@@ -9,6 +12,7 @@ class QnA extends React.Component {
       questionSearch: ''
     }
     this.changeQuestion = this.changeQuestion.bind(this);
+    this.onSubmit = this.onSubmit.bind(this);
   }
 
   changeQuestion(event) {
```
Comment (PBandJaimie) [inline client/src/components/QandA/QandA.jsx:18]: Is this missing a closing bracket? Or am I just not seeing it?

File: client/src/components/QandA/SearchQuestion.jsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+const SearchQuestion = (props) => {
+  return (
+    <div className='form-div'>
+      <form onSubmit={props.submit}>
+        <input type = 'text'
+          placeholder='HAVE A QUESTION? SEARCH FOR ANSWERS...'
+          onChange={props.cQuestion}
+          value={props.questionSearch}
```
Comment (PBandJaimie) [inline client/src/components/QandA/SearchQuestion.jsx:10]: I believe this should be value={props.qSearch}

File: server/routes/qnaRoutes.js [hunk 3]
```diff
@@ -5,15 +5,17 @@ const axios = require('axios');
 const config = require('../../config.js');
 
 router.get('/questions', (req, res) => {
-  let id = Number(req.data.id);
+  let id = Number(28212);
 
-  axios.get(`https://app-hrsei-api.herokuapp.com/api/fec2/hr-rpp/qa/questions?product_id=${id}&page=1&count=100`, {
+  //&page=1&count=100
```
Comment (PBandJaimie) [inline server/routes/qnaRoutes.js:10]: Do you need this comment? If so, cool!

**Review by PBandJaimie — CHANGES_REQUESTED** [review]
@a-nouraldean Wow! Your code is incredibly clean. Well done!!! I requested a change or two, but if you don't feel they are necessary, just let me know and I'll approve it!

File: server/routes/qnaRoutes.js (hunk 3 above)
Comment (a-nouraldean) [reply server/routes/qnaRoutes.js:10]: Yeah I do. I don't want to limit in the server request. Can manage response in front rend.

File: client/src/components/QandA/SearchQuestion.jsx (hunk 2 above)
Comment (a-nouraldean) [reply client/src/components/QandA/SearchQuestion.jsx:10]: You are right! Thank!

File: client/src/components/QandA/QandA.jsx (hunk 1 above)
Comment (a-nouraldean) [reply client/src/components/QandA/QandA.jsx:18]: No! These are part of of the the constructor.

File: client/src/components/QandA/SearchQuestion.jsx (hunk 2 above)
Comment (a-nouraldean) [reply client/src/components/QandA/SearchQuestion.jsx:10]: I have updated. We can merge this pull request for now as it is not impacting operations

**Pull request merged**